docs/content/en/docs/user-guide/configuration-reference.md

```diff
@@ -27,7 +27,7 @@ spec:
 | workloads | [][KubernetesWorkload](/docs/user-guide/configuration-reference/#kubernetesworkload) | Which Kubernetes resources should be considered as the Workloads of application. Empty means all Deployment resources. | No |
 | trafficRouting | [KubernetesTrafficRouting](/docs/user-guide/configuration-reference/#kubernetestrafficrouting) | How to change traffic routing percentages. | No |
 | sealedSecrets | [][SealedSecretMapping](/docs/user-guide/configuration-reference/#sealedsecretmapping) | The list of sealed secrets should be decrypted. | No |
-<!-- | dependencies | []string | List of directories where their changes will trigger the deployment. | No | -->
+| changes | []string | List of directories or files where their changes will trigger the deployment. Regular expression can be used. | No |
```

Comment:
I renamed this because we may have a "requires/dependencies" feature in the future (a deployment depends on another).
Please tell me if you have a better name.

Reply:
how about `watchingPaths` since this field is about the paths which piped watches for changes on that 👀

Reply:
Nice name. Thank you.

Reply:
@nakabonne How do you feel about that name?

Reply:
Started thinking just now...

Reply:
It'd better clarify what it does. So:
- triggerPaths
- triggerSources

would be good.

Reply:
Nice. Let's pick "triggerPaths".

Reply:
then it should be `triggeredPaths`, I guess 😂